js/script.js

```diff
@@ -787,10 +787,27 @@ function Add_EventHandlers_To_Document()
     })
 }
 
+function selectPalette() {
```

Comment:
A little ⛏,  but I'd prefer we stick to the `This_Is_My_Function_Name` for now for consistency.

Looking back at this code, and having read that folks don't like this convention (I like it personally), I'm tempted to create an issue to change to `thisStyle`

Reply:
I've seen the convention go both way personally. I learned the style of: thisStyle first, so it's what I habitually do. But I agree, consistency is important, so if you prefer a different scheme - then I'm happy to oblige. You're the project owner after all.

Reply:
And same here, I've seen `thisStyle` waay more than `This_Style` (I do forget where I read it but this one blogpost/style guide really didn't like `This_Style` 😂 ... still not sure if there is a deeper reason, but I like it for clarity ... `thisIsAFunction` gets hard to read personally when the sub works are <3 characters long ya know?)